client/app/scripts/superdesk-search/views/item-search.html

```diff
@@ -88,6 +88,30 @@
                 <div sd-meta-terms data-item="subjectitems" data-field="subject" data-unique="qcode" data-list="subjectcodes" data-change="subjectSearch(subjectitems)"></div>
             </div>
         </div>
+        <div class="field">
+            <label for="from-desk">
+                {{:: 'From Desk' | translate}}
+            </label>
+            <select
+                id="from-desk" name="from-desk"
+                ng-options="d._id as d.name for d in desks._items track by d._id"
```

Comment:
As per [ng-options docs](https://docs.angularjs.org/api/ng/directive/ngOptions), the `select` and `track by` parts of the expression are not designed to work together.
It is not an issue in this case, since they are the same, but it could become if somebody later makes changes to it. It's thus probably best to simply remove the `track by` part.